content/en/docs/concepts/security/multi-tenancy.md

```diff
@@ -462,6 +462,7 @@ listed below.
 
 #### Multi-team tenancy
 
+* [Multi Tenant Operator](https://docs.stakater.com/mto/)
 * [Capsule](https://github.com/clastix/capsule)
```

Comment:
Hi. These need to be in alphabetical order
```suggestion
{{% thirdparty-content %}}

* [Capsule](https://github.com/clastix/capsule)
* [Multi Tenant Operator](https://docs.stakater.com/mto/)
```
(the `thirdparty-content` shortcode is already in use on the page, but I recommend repeating it)